packages/xrpc-server/src/types.ts

```diff
@@ -12,6 +12,12 @@ export const handlerInput = zod.object({
 })
 export type HandlerInput = zod.infer<typeof handlerInput>
 
+export const handlerAuth = zod.object({
+  credentials: zod.any(),
```

Comment:
could we use generics here so we actually know what types we're getting out of the auth fn?

Reply:
In the handlers inside the pds, each handler's `ctx.auth` is properly typed.  That level of specificity is provided by lex-cli's typings rather than xrpc-server's.  That's the same way the handler `input` and `params` typings work, too.  Do you think that covers it?

Reply:
oh yup that's good 👍